nemo/collections/nlp/modules/common/megatron/language_model.py

```diff
@@ -178,7 +178,7 @@ def forward(self, hidden_states, sequence_index=0):
         # gather data along sequence dimensions
         # same pooler is run on all tensor parallel nodes
         if self.sequence_parallel:
-            hidden_states = tensor_parallel.gather_from_sequence_parallel_region()
+            hidden_states = tensor_parallel.mappings.gather_from_sequence_parallel_region(hidden_states)
```

Comment:
@yidong72  Make sure this is okay

Reply:
looks good. please verify the shape before and after gather is the expected